File: js_func/handlers/handler.js
// Метод handler. Данный метод будет обрабатывать HTTP запросы поступающие к функции
// Принимает объекты запроса Request ssи Response библиотеки Express (http://expressjs.com)
exports.handler = (req, res) => {
    // Логирование входящего запроса
    console.log(`Request received: ${JSON.stringify(req.body)}\nMethod: ${req.method}`);

    // Подготовка и возврат ответа на вызов
    res.status(200).send(`Hello from NodeJS function!\nYou said: ${JSON.stringify(req.body)}`);
};

File: go_func/handlers/handler.go
package handlers

import (
	"fmt"
	"io/ioutil"
	"log"
	"net/http"
)

// Метод Handler. Данный метод будет обрабатывать HTTP запросы поступающие к функции
func Handler(w http.ResponseWriter, r *http.Request) {
	if r.Body != nil {
		defer r.Body.Close()
	}

	// Чтение тела запроса
	body, _ := ioutil.ReadAll(r.Body)

	// Логирование входящего запроса
	log.Printf("Request received: %s\nMethod: %s", string(body), r.Method)

	// Подготовка и возврат ответа на вызов
	w.WriteHeader(http.StatusOK)
	w.Write([]byte(fmt.Sprintf("Hello from Go function!\nYou said: %s", string(body))))
}
